TESTS/usb_device/basic/USBTester.cpp

```diff
@@ -174,7 +177,7 @@ void USBTester::callback_request(const setup_packet_t *setup)
         }
     }
 
-    if (delay) {
+    if (delay != std::chrono::duration<int, std::milli>::zero()) {
```

Comment:
If you want to use the method, better to just say `delay.zero()`. But you may as well just say `0s` in normal life, I think. Using the `zero()` method is only really worth it where it's important to maintain the specific type.

Yes `0s` is 64-bit, so it does make it a `!=` comparison of a 32-bit millisecond count to a 64-bit zero, but the compiler should, I hope, realise it doesn't actually need to do any 64-bit work for that. Maybe you could check if `!= 0s` is bigger than `!= delay.zero()` on each toolchain? Put my mind at rest.

(But as this is test code, I wouldn't even sweat 32-bit/64-bit anyway. I'd have just made delay be `milliseconds` (ie 64-bit) to save typing.)

And you already used `0s` above when you initialised it, so you're not being consistent!